sdk/appconfiguration/app-configuration/test/etags.spec.ts

```diff
@@ -69,23 +68,27 @@ describe("etags", () => {
     );
   });
 
-  it("get using ifNoneMatch to only get the setting if it's changed", async () => {
+  it("get using ifNoneMatch to only get the setting if it's changed (ie: safe GET)", async () => {
     const originalSetting = await client.setConfigurationSetting({
       key: key,
       value: "world"
     });
 
     // only get the setting if it changed (it hasn't)
-    try {
-      await client.getConfigurationSetting({ key }, {
-        ifNoneMatch: originalSetting.etag
-      });
-    } catch (err) {
-      assert.ok(err instanceof ResponseBodyNotFoundError);
-      assert.equal("The requested setting's value has not changed since the last request.", err.message);
-      assert.equal("Resource same as remote", err.code);
-      assert.equal(304, err.statusCode);
-    }
+    const response = await client.getConfigurationSetting({ key }, {
+      ifNoneMatch: originalSetting.etag
+    });
+
+    assert.equal(response._response.status, 304);
```

Comment:
Can status be lifted to a property of the response? `response._response` is a bridge too far for me...

Reply:
I'll hoist it up but as 'statusCode' since 'status' seems too prone to name collision.